src/main/scala/net/ripe/rpki/publicationserver/fs/RrdpRepositoryWriter.scala

```diff
@@ -55,9 +55,13 @@ class RrdpRepositoryWriter extends Logging {
   }
 
   def deleteDelta(rootDir: String, sessionId: UUID, serial: Long): AnyVal = {
-    deleteSessionFile(rootDir, sessionId, serial, Rrdp.deltaFilename)
+    val sessionSerialDir = Paths.get(rootDir, sessionId.toString, serial.toString)
```

Comment:
old `deleteSeessionFile` is now unused, maybe this is its new impementation.

Reply:
Indeed!